lib/pact/init/cli_provider.rb

```diff
@@ -0,0 +1,25 @@
+require 'thor'
+require 'pact/init'
+
+module Pact
+  module Init
+    class CLIProvider < Thor
+
+      desc 'create' , "initalize your spec dir with the helper class"
+      method_option :consumer, aliases: "--consumer", :desc => "The name of your consumer"
+      method_option :provider, aliases: "--provider", :desc => "The name of your provider"
+
+      def create(args = {})
+        if !options['consumer'].nil?
+          args[:consumer] = options['consumer']
```

Comment:
This works fine, but if you're interested in learning the idiomatic ruby way, you would typically do `if options['consumer']`